Random Users/View Controllers/PersonDetailViewController.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  PersonDetailViewController.swift
+//  Random Users
+//
+//  Created by Marc Jacques on 5/17/20.
+//  Copyright © 2019 Erica Sadun. All rights reserved.
+//
+
+import UIKit
+
+class PersonDetailViewController: UIViewController {
+
+    @IBOutlet weak var imageView: UIImageView!
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var emailLabel: UILabel!
+    @IBOutlet weak var phoneLabel: UILabel!
+    
+    
+    var person: Person?
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setViews()
+    }
+    
+    private func setViews() {
+        guard let person = person,
+        isViewLoaded == true else {return}
```

Comment:
You can just call isViewLoaded on its own. No need to set == true. Setting anything before an else automatically checks if it is true.